src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -273,7 +275,8 @@ private void testEditQuestionAction() throws Exception {
         
         //change back
         feedbackEditPage.clickVisibilityOptionsForQuestion1();
-        feedbackEditPage.selectGiverTypeForQuestion1("Me (Session creator)");
+        feedbackEditPage.enableOtherFeedbackPathOptions(1);
+        feedbackEditPage.selectGiverTypeForQuestion1("Feedback session creator (i.e., me)");
```

Comment:
I'm quite sure you've migrated all `*ForQuestion1(...)` functions to something else, right?